app/src/main/res/drawable/on_boarding_develop.xml

```diff
@@ -0,0 +1,109 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="360dp"
```

Comment:
We should avoid using .svg files for images having size greater than 200dp. 
The benefit of using svg over png gets lost in this case.
Better to use webp format for this